### ISSUE TITLE
Don't work me

### ISSUE THREAD

**Issue by alextomas80** [beyonk-adventures/svelte-google-analytics#4]
Hi! 

I have followed the steps in the example, but it doesn't work for me. I don't see the page views in google analytics.

![image](https://user-images.githubusercontent.com/60695705/105537374-2060fc00-5cf2-11eb-996e-2b428fb5961c.png)

On the other hand, **the events** do work for me as in the example.

![image](https://user-images.githubusercontent.com/60695705/105537433-35d62600-5cf2-11eb-97fd-6c253860f63f.png)

I hope you can help me.

**Comment by oskarhane** [comment]
You might need to send the events manually on every transition.

In Sapper, this is what I do in `_layout.svelte`

```
<script>
import { stores } from "@sapper/app";
import { GoogleAnalytics, ga } from "@beyonk/svelte-google-analytics";

const { page } = stores();

onMount(() => {
        return page.subscribe(({ path }) => {
            ga.addEvent("page_view", { page_path: path });
        });
    });
</script>

<GoogleAnalytics properties={['UA-XXXX-X', {send_page_view: false}]} />
```

Notice the `{send_page_view: false}`, which I don't think works because we always get two page view on the initial page load.

Is there a way to define configuration options @antony?

**Comment by antony** [comment]
@alextomas80 I've not tried it with Routify, but this library doesn't deal with recording page views, that's down to Google Analytics, which claims to do it by default. It might br worth using the GA debugging tools to find out what is/isn't being recorded.

@oskarhane `properties` is for google analytics property ids - I've not tested it by passing in config options, but this would probably work.

I think in order to properly support configuration a PR would be ideal where we could pass the property configuration optionally as an object perhaps, with the property id as the key, and config being the value.

I'm not going to have time to do this myself, but happy to accept a PR.

**Comment by antony** [comment]
Fixed by v2.1.0
